RecoTauTag/RecoTau/interface/DeepTauBase.h

```diff
@@ -35,12 +65,10 @@ class DeepTauBase : public edm::stream::EDProducer<> {
     using ElectronCollection = pat::ElectronCollection;
     using MuonCollection = pat::MuonCollection;
     using LorentzVectorXYZ = ROOT::Math::LorentzVector<ROOT::Math::PxPyPzE4D<double>>;
-    using GraphPtr = std::shared_ptr<tensorflow::GraphDef>;
-    using Cutter = StringObjectFunction<TauType>;
+    using Cutter = TauWPThreshold;
```

Comment:
@kandrosov  introduced this changes in the commit ebae55398568db2041060224c6f57e9874058c8c, as he says: 

> It is needed because the use of the standard
StringObjectFunction class to parse complex expression results in an
extensive memory usage (> 100 MB per expression)

Reply:
OK, I see. However, I prefer to use common string parser than homemade one. I propose to ask in how to proceed in the PR to official CMSSW. 
BTW, I think that one can simplify the cut expression for Tight DPFv0, but it will be even less human-friendly...

Reply:
In my opinion, human readability in the configuration is the most important criterion. This parser can cover all our needs easily and allows to define the cut in c++ without any limitations that exist in the "common" parser. And, personally, I find the implementation of the parser within CMSSW to be an old hardly readable code, which contains one patch on to of the other and which is a legacy of pre-root 6 era. I don't see any reason to use it for this particular case.